src/loaders/api/api.js

```diff
@@ -82,14 +141,14 @@ export function setAPI (agentIdentifier, forceDrain) {
     }
   }
 
-  mapOwn('actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get'.split(','), function addApi (n, name) {
+  void ['actionText', 'setName', 'setAttribute', 'save', 'ignore', 'onEnd', 'getContext', 'end', 'get'].forEach(name => {
```

Comment:
I don't think the `void` operator is necessary here.

Reply:
Eslint 🙃